code-challenges.test.js

```diff
@@ -13,19 +13,67 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("fibonacci", () => {
+  it("returns a an array of fibonacci numbers the same length of the input", () => {
+    expect(fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibonacci(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
 
 const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+//Pseudocode:
+//Input:A number that is greater than 2
+//output:An array that is the same length of the input number.
+//create a function called fibonnaci that takes in a number.
+//create a variable in function with an empty array to hold numbers generated that starts a [1,1] like every fibonacci sequence.
+//create a for loop that iterates through the newArray starting at the second index though the value of the number input in increments of 1.
+//use .push to move the added two previous numbers into the newArray.
```

Comment:
To help you be a bit more descriptive in your pseudocode, you can reference that you intend to do that by using the index - 1 & -2 to get those previous numbers